unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -38,6 +39,9 @@ class TimeTest {
   }
 
   @Test def localtimeShouldTransformTheEpochToLocaltime(): Unit = {
+    assumeFalse(
```

Comment:
Again, I am probably in the minority.  I dislike `assumeFalse` 
__intensely__ because it throws scary looking garbage into the 
log file. The garbage looks like an error, but is a bit of 'known error'
lore which must be re-parsed and re-learned after one has been
away for a month or so.

To me, a simple `if` which executes only if not FreeBSD conveys the
intent and does not generate `known error` garbage.  There is nothing
to warn about, the test is executing successfully on both FreeBSD and,
say, Ubuntu.  It is just on the former, the code sees that it has nothing to
do. There is no exceptional (with a small e) code flow to notify 
someone reading the log files about.

I spend a __lot__ of time reading SN logs, and so am somewhat sensitive
about wasting time skipping over "known errors' that I used to know.

Again, I may be in the minority, so you should probably see what a 
Scala Center person advises before making any change.

Reply:
And yes, I would leave a one line comment "FreeBSD does not have the 'timezone' variable".  Some poor soul down the line trying to read and
comprehend this stuff will sing your praises (which is why, I assume,
you put such a nice comment in the assumeFalse).

Reply:
I'm not sure to agree on this one. I've added the `assume` on purpose exactly to not hide that FreeBSD lacks the timezone variable. The warning will show up only on FreeBSD and makes clear that you cannot rely on the timezone variable in your code if it's run on FreeBSD.

Reply:
Yes, I believe this is how we use `assume`, precisely to highlight the assumptions for test coverage.